src/lib.rs

```diff
@@ -132,9 +132,17 @@ struct CommonOptions {
     #[structopt(long)]
     jitdump: bool,
 
-    /// Run optimization passes on translated functions
+    /// Run optimization passes on translated functions, on by default
     #[structopt(short = "O", long)]
     optimize: bool,
+
+    /// Optimization level for generated functions (0, 1, 2 (default), or s)
```

Comment:
Should we note what `s` stands for here in the CLI help?

Reply:
Perhaps an additional mention that `0` indicates no optimizations?

Reply:
It looked like `--help` didn't render multiline doc comments here so I tried to squish this into the line, bug sounds good!